tests/api_v2/test_util_methods.py

```diff
@@ -1,23 +1,21 @@
 # Copyright (c) 2017 Shotgun Software Inc.
-# 
+#
 # CONFIDENTIAL AND PROPRIETARY
-# 
-# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
 # Source Code License included in this distribution package. See LICENSE.
-# By accessing, using, copying or modifying this work you indicate your 
-# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 import os
 import sys
 
-from tank_test.tank_test_base import setUpModule
+from tank_test.tank_test_base import setUpModule # noqa
 
-# import the test base class
```

Comment:
tk-core's `run_test.py` automatically adds the unit test's `tests/python` submodule to `sys.path`